code/modules/mob/living/carbon/human/human_examine.dm

```diff
@@ -55,7 +55,7 @@
 	if(skip_jumpsuit && skip_face || HAS_TRAIT(src, TRAIT_NOEXAMINE)) //either obscured or on the nospecies list
 		msg += "!"    //omit the species when examining
 	else if(displayed_species == "Slime People") //snowflakey because Slime People are defined as a plural
```

Comment:
Code looks good, but while you're here, I think it would be better to refactor this so sometime someone changes line 60,  they dont need to change this one too.
```suggestion
	else
		var/species_name =  lowertext(displayed_species)
		if(displayed_species == "Slime People") //snowflakey because Slime People are defined as a plural
			species_name = "slime person"
		msg += ", \a [height]<b><font color='[examine_color]'> [species_name]</font></b> with \a [physique] physique!"
```